patternfly/patternfly-elements
# feat(tools): parameterize react wrapper generator

Closes #2649 

## What I did

1. parameterize react wrapper gen

## Testing Instructions

0. build react wrappers and compare output to main
1. see [rhds pr](https://github.com/RedHat-UX/red-hat-design-system/pull/1249)
